cli/internal/fs/turbo_json_test.go

```diff
@@ -76,3 +77,35 @@ func Test_ReadTurboJSON(t *testing.T) {
 	}
 	assert.EqualValues(t, remoteCacheOptionsExpected, turboJSON.RemoteCacheOptions)
 }
+
+func Test_ValidateTurboJson(t *testing.T) {
```

Comment:
I based this testing pattern on what seems to be relatively common in the codebase. Is this a common testing pattern in the Go ecosystem, as opposed to having many granular test functions?